### PR TITLE
Add single csv values so that there is a single response header added…

### PR DESCRIPTION
**Pull request by harshgMSFT**
… per value.

### CODE REVIEW

**Comment by harshgMSFT** [comment]
@Tratcher

File: src/Microsoft.AspNet.Cors.Core/CorsService.cs [hunk 1]
```diff
@@ -137,7 +137,9 @@ public virtual void ApplyResult(CorsResult result, HttpResponse response)
 
                 if (nonSimpleAllowMethods.Length > 0)
                 {
-                    headers.Add(CorsConstants.AccessControlAllowMethods, nonSimpleAllowMethods);
+                    headers.SetCommaSeparatedValues(
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Cors.Core/CorsService.cs]: If you want them to end up as separate headers you need to call SetValues:
https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNet.Http.Abstractions/IHeaderDictionary.cs#L75

Comment (Tratcher) [inline src/Microsoft.AspNet.Cors.Core/CorsService.cs]: Nevermind, I was remembering the issue backwards.

**Comment by Tratcher** [comment]
#22 :shipit:

**Pull request merged**